charts/px-backup/templates/px-central-ui/pxcentral-ui.yaml

```diff
@@ -46,7 +46,7 @@ data:
         proxy_set_header X-Forwarded-Proto $myscheme;
         proxy_set_header X-Forwarded-Host  $http_host;
         proxy_set_header Port  $http_port;
-        proxy_set_header X-Forwarded-Port  $server_port;
+        proxy_set_header X-Forwarded-Port  $http_port;
```

Comment:
do you need it for both lhbackend and auth? or just auth?

Reply:
Yes, we need this in both the places.